### PR TITLE
After login, Displaying folders and images for user to select

### PR DESCRIPTION
**Pull request by RishikaBhatia17**
In the tool, after selecting the "choose photos from google drive" button, images are displayed in a random order and this pull request fixes this issue. Folders are first displayed after selecting "choose photos from google drive" button as shown below.

image

Then we can navigate through folders and view files and sub folders as shown below

image

To view only images after navigating to a folder, select file type as images from advanced search options as shown below

image

image

Then select the images to upload to Wikimedia commons. We can select multiple images randomly by selecting the images while pressing the CTRL button as shown below. This Fixes tonythomas01/gdrive-to-commons-repo#34

image

After this we can upload the images to Wikimedia commons.

Fixes tonythomas01/gdrive-to-commons-repo#33

### CODE REVIEW

File: uploader/templates/upload.html [hunk 1]
```diff
@@ -271,8 +271,8 @@ <h5 class="card-title text-center">
   // Create and render a Picker object for searching images.
   function createPicker() {
     if (pickerApiLoaded && oauthToken) {
-      var view = new google.picker.View(google.picker.ViewId.DOCS);
-      view.setMimeTypes("image/png,image/jpeg,image/jpg");
+      var view = new google.picker.DocsView().setIncludeFolders(true);
+      view.setMimeTypes("");
```
Comment (tonythomas01) [inline uploader/templates/upload.html]: Will this result in people uploading non image files ? We should not allow them to see those btw.

Comment (RishikaBhatia17) [reply uploader/templates/upload.html]: @tonythomas01 The required changes are done and now the user can only view images and no other files. I request you to review the changes and merge the pull request.
Thanks!

**Comment by tonythomas01** [comment]
I am sorry, but when I login with my Google account, I see a lot of folders (mostly inner folders deep down) and kind of am missing the real folders I see when I open Google Drive. Is it the same with you ?

**Comment by RishikaBhatia17** [comment]
@tonythomas01 I have made changes such that folders are seen in the same the real folders are seen when google drive is opened. Now the Select file UI has become same like the UI of our google drive. I request you to review the changes and merge the pull requests.
Thank You!

**Comment by tonythomas01** [comment]
Works. Thank you!

**Pull request merged**